packages/std/src/amp/addresses.rs

```diff
@@ -195,8 +195,9 @@ impl AndrAddr {
     }
     /// Gets the raw path for a given AndrAddr by stripping away any protocols or chain declarations.
     ///
-    /// E.g. `ibc://cosmoshub-4/user/app/component` would return `/user/app/component`
-    /// E.g. `ibc://cosmoshub-4/user` would return `user`
+    /// E.g. `ibc://cosmoshub-4/user/app/component` would return `/home/app/component`
+    /// E.g. `ibc://cosmoshub-4/cosmos1...` would return `cosmos1...`
+    /// E.g. `ibc://chain/ibc://chain2/home/app/component` would return `ibc://chain2/home/app/component`
```

Comment:
Oh this is a neat idea. I wonder if there's a cleaner way to achieve this though

Reply:
Let's remove this capability for now and consider an alternative way to achieve this.